src/bidiMapper/domains/context/browsingContextProcessor.ts

```diff
@@ -286,46 +317,72 @@ export class BrowsingContextProcessor {
   async process_script_addPreloadScript(
     params: Script.AddPreloadScriptParameters
   ): Promise<Script.AddPreloadScriptResult> {
-    const contexts: BrowsingContextImpl[] = [];
-    const scripts: Script.PreloadScript[] = [];
+    const cdpTargets = new Set<CdpTarget>(
```

Comment:
If set doesn't work well with instances, you can rely on `cdpTarget.cdpSessionId`.